app/_locales/hi/messages.json

```diff
@@ -3171,10 +3174,10 @@
     "message": "स्मार्ट लेनदेन अनुपलब्ध होने पर भी आप अपने टोकनों को स्वैप कर सकते हैं।"
   },
   "stxPendingPrivatelySubmittingSwap": {
-    "message": "आपका स्वैप निजी रूप से भेजा जा रहा है..."
+    "message": "आपका स्वैप निजी रूप से सबमिट किया जा रहा है..."
```

Comment:
In the English translations, we capitalize "Swap" in these cases. Should we spell out "Swap" for:
- `stxPendingPrivatelySubmittingSwap`
- `stxPendingPubliclySubmittingSwap`
- `stxSwapCompleteIn`
?

```
  "stxPendingPrivatelySubmittingSwap": {
    "message": "Privately submitting your Swap..."
  },
  "stxPendingPubliclySubmittingSwap": {
    "message": "Publicly submitting your Swap..."
  },

  // .. 

  "stxSwapCompleteIn": {
    "message": "Swap will complete in <",
    "description": "'<' means 'less than', e.g. Swap will complete in < 2:59"
  },
```

Reply:
We confirmed that we would want to preserve "swap", "swaps", "Swap", and "Swaps" in our translations:
https://consensys.slack.com/archives/GN3SR3GNM/p1655931775407429 (internal)

@adonesky1 is working on implementing this request with Crowdin.

I think we can resolve this in a future PR. This request should not be a blocker for this PR.

Reply:
@digiwand are you sure we want to preserve `swaps` (swap, Swap, Swaps), the conversation you reference seems to confirm that we want to preserve `snaps`?

Reply:
Hello, yes, based on this Slack internal slack conversation: https://consensys.slack.com/archives/GN3SR3GNM/p1655931775407429

Reply:
Hi @adonesky1, 
Yes, based on this Slack internal slack conversation: https://consensys.slack.com/archives/GN3SR3GNM/p1655931775407429

Reply:
oops... sorry... let me try this again. BRB

Reply:
@adonesky1  oops, got the "n" and the "w" mixed up 😅 sorry please ignore this thread now! For Swaps, I've verified that translating this word is okay because Uniswap also translates "swaps" on their page 

e.g. 
https://app.uniswap.org/#/swap?chain=mainnet&lng=de-DE